src/MSBuild/Resources/Strings.resx

```diff
@@ -722,6 +722,22 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
     </comment>
   </data>
+  <data name="HelpMessage_35_GraphBuildSwitch" UESanitized="false" Visibility="Public">
+    <value>/graphBuild
+                     Causes MSBuild to construct a project graph and build it
+                     from the bottom up as opposed to discovering and building
```

Comment:
I don't like the phrase "bottom up", but we have plenty of time to tweak this :)